content/english/python-tensorflow/plotting_model.md

```diff
@@ -7,20 +7,27 @@ weight: 11
 draft: false
 ---
 
-Now that we are able to classify the category of an individual piece of clothing using your model, we will now visualize these predictions by creating a bar plot for a specified image. 
-
 <iframe width="100%" height="600px" src="https://www.youtube.com/embed/FrO66fkSU7M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Copy the following code into Colab:
+</br>
+</br>
+
+Now that we are able to classify the category of an individual piece of clothing using our model, we will visualize these predictions by creating a bar plot for a specified image. 
 
-{{% notice note %}}
+## Predicated Classification 
 
 The `plot_image` function displays an image along with its predicted classification, the confidence level of the model for that category, and the actual classification (which is for reference).
 
-The `plot_value_array` function plots the confidence values generated by the model as a bar graph.
+{{% notice tip %}}
+A confidence level is used to convey how confident or certain we are in our answer being correct. If you have to predict whether it will rain tomorrow, your possible answers are yes or no. What if you're not totally certain in your answer, though? If the weather man predicted rain for tomorrow, you may say you are 90% confident. This means that you are pretty sure it will rain tomorrow, but it also tells us that you think there is a 10% chance you may be wrong. 
```

Comment:
```suggestion
A **confidence level** is used to convey how confident or certain we are in our answer being correct. If you have to predict whether it will rain tomorrow, your possible answers are yes or no. What if you're not totally certain in your answer, though? If the weather reporter predicted rain for tomorrow, you may say you are 90% confident. This means that you are pretty sure it will rain tomorrow, but it also tells us that you think there is a 10% chance you may be wrong. 
```